.envrc.example

```diff
@@ -24,7 +25,11 @@ export T_INTERMINE_URL=https://targetmine.mizuguchilab.org
 
 # Additional back end variables for FusionAuth
 # (the values below work out of the box with the docker-compose file)
+
+#External URL that the user can be redirected to
 export FUSIONAUTH_BASEURL="http://localhost:9011"
+#Internal URL for connecting to fusionauth.
+export FUSIONAUTH_APP_URL="http://fusionauth:9011"
```

Comment:
This should point to localhost, rather than fusionauth, to be consistent with lines 13-14. A comment similar to line 12 would be a good idea too.